test/fixtures/api/responses/get-balance-sheet.json

```diff
@@ -0,0 +1,68 @@
+{
+  "balances": [
+    {
+      "counterparty": "rKm4uWpg9tfwbVSeATv4KxDe6mpE9yPkgJ",
+      "balances": [
```

Comment:
How about making `balances` and object keyed by the counterparty address to avoid the redundant `balances` (and similar for assets)?